installing/installing_bare_metal/installing-bare-metal.adoc

```diff
@@ -49,6 +49,8 @@ include::modules/installation-initializing-manual.adoc[leveloffset=+1]
 
 include::modules/installation-bare-metal-config-yaml.adoc[leveloffset=+2]
 
+include::modules/installation-configure-proxy.adoc[leveloffset=+2]
```

Comment:
Take a look at slide 20 from the presentation that I shared with you regarding bare-metal installs. The cluster network operator uses `machineCIDR` from install-config for noProxy status. Node IP's must be from machineCIDR for bare-metal installs. Otherwise, the node IP's/CIDR must be added manually to `noProxy` of the proxy object or install-config. For example:

```
$ oc get cm/cluster-config-v1 -n kube-system -o yaml | grep machineCIDR
      machineCIDR: 10.0.0.0/16
$ oc get proxy/cluster -o yaml | grep 10.0.0.0/16
  noProxy: .cluster.local,.svc,.us-west-2.compute.internal,10.0.0.0/16,10.128.0.0/14,127.0.0.1,169.254.169.254,172.30.0.0/16,api-int.eric-proxy.devcluster.openshift.com,api.eric-proxy.devcluster.openshift.com,apps.eric-proxy.devcluster.openshift.com,etcd-0.eric-proxy.devcluster.openshift.com,etcd-1.eric-proxy.devcluster.openshift.com,etcd-2.eric-proxy.devcluster.openshift.com,localhost
```

Reply:
@gpei @danehans I've asked a question in the BZ here if either of you have input as well: https://bugzilla.redhat.com/show_bug.cgi?id=1752832#c2

Basically just wanting to confirm if bare metal is still the only place this note needs to be during installation, and then for scaling workers after installation, does that include both manual scaling and autoscaling? And if so for autoscaling, how would that work?